src/accounts.ts

```
@@ -0,0 +1,175 @@
+      ) as RawRegistry;
+      const list: RawAccount[] = Array.isArray(raw?.accounts)
+        ? raw.accounts
+        : [];
+      accounts = list.map((a: RawAccount): CodevAccount => {
+        const dir = expandHome(
+          String(a.dir || path.join(os.homedir(), '.claude')),
+        );
+        // Tolerate a partial registry entry: if isDefault is omitted, infer it
+        // from the top-level defaultAccount label.
+        const isDefault =
+          a.isDefault ?? (!!raw.defaultAccount && a.label === raw.defaultAccount);
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)